examples/c/ex_col_store.c

```diff
@@ -0,0 +1,555 @@
+#include <test_util.h>
```

Comment:
Please add a copyright 2020 notice. (Later WT-6926 will update it to 2021 notice.)